src/components/GGanttColumn.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div class="g-gantt-column" :style="{ fontFamily: font, color: colors.text }">
+    <span class="g-gantt-header" :style="{ background: colors.primary }">{{ columnTitle }}</span>
+    <div class="g-gantt-labels">
+      <div v-for="(label, index) in labels" :key="index" class="g-gantt-label" :style="{
+        background: index % 2 === 0 ? colors.ternary : colors.quartenary,
+        height: `${rowHeight}px`,
+      }">
+        <span>{{ label }}</span>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import provideConfig from "../provider/provideConfig.js"
+
+const { font, colors, labels, columnTitle, rowHeight } = provideConfig()
+</script>
+
+<style>
+.g-gantt-column {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  color: rgb(64, 64, 64);
+  justify-content: flex-start;
```

Comment:
I'd center this both horizontally as well as vertically:
`justify-content: center`

Reply:
adjusted